data_steward/tools/clean_fitbit.sh

```diff
@@ -78,24 +77,24 @@ export BIGQUERY_DATASET_ID="${fitbit_dataset}"
-#bq mk --transfer_config --project_id="${APP_ID}" --data_source="cross_region_copy" --target_dataset="${registered_fitbit_deid}" --display_name='Create Fitbit Deid' --params="${transfer_params}"
+#transfer_params='{"source_dataset_id":"'${fitbit_dataset}'","source_project_id":"'${APP_ID}'"'
+#bq mk --transfer_config --project_id="${APP_ID}" --data_source="cross_region_copy" --target_dataset="${fitbit_deid_dataset}" --display_name='Create Fitbit Deid' --params="${transfer_params}"
 
 # create empty fitbit sandbox dataset
-sandbox_dataset="${registered_fitbit_deid}_sandbox"
-bq mk --dataset --description "Sandbox created for storing records affected by the cleaning rules applied to ${registered_fitbit_deid}" --label "phase:sandbox" --label "release_tag:${dataset_release_tag}" --label "de_identified:false" "${APP_ID}":"${sandbox_dataset}"
+sandbox_dataset="${fitbit_deid_dataset}_sandbox"
+bq mk --dataset --description "Sandbox created for storing records affected by the cleaning rules applied to ${fitbit_deid_dataset}" --label "phase:sandbox" --label "release_tag:${dataset_release_tag}" --label "de_identified:false" "${APP_ID}":"${sandbox_dataset}"
 
 # Create logs dir
 LOGS_DIR="${DATA_STEWARD_DIR}/logs"
 mkdir -p "${LOGS_DIR}"
 
 # Apply cleaning rules
-python "${CLEAN_DEID_DIR}/remove_fitbit_data_if_max_age_exceeded.py" --project_id "${APP_ID}" --dataset_id "${registered_fitbit_deid}" --sandbox_dataset_id "${sandbox_dataset}" --combined_dataset_id "${combined_dataset}" -s 2>&1 | tee -a "${LOGS_DIR}"/fitbit_log.txt
-python "${CLEAN_DEID_DIR}/pid_rid_map.py" --project_id "${APP_ID}" --dataset_id "${registered_fitbit_deid}" --sandbox_dataset_id "${sandbox_dataset}" --mapping_dataset_id "${mapping_dataset}" --mapping_table_id "${mapping_table}" -s 2>&1 | tee -a "${LOGS_DIR}"/fitbit_log.txt
-python "${CLEAN_DEID_DIR}/fitbit_dateshift.py" --project_id "${APP_ID}" --dataset_id "${registered_fitbit_deid}" --sandbox_dataset_id "${sandbox_dataset}" --mapping_dataset_id "${mapping_dataset}" --mapping_table_id "${mapping_table}" -s 2>&1 | tee -a "${LOGS_DIR}"/fitbit_log.txt
+python "${CLEAN_DEID_DIR}/remove_fitbit_data_if_max_age_exceeded.py" --project_id "${APP_ID}" --dataset_id "${fitbit_deid_dataset}" --sandbox_dataset_id "${sandbox_dataset}" --combined_dataset_id "${combined_dataset}" -s 2>&1 | tee -a "${LOGS_DIR}"/fitbit_log.txt
+python "${CLEAN_DEID_DIR}/pid_rid_map.py" --project_id "${APP_ID}" --dataset_id "${fitbit_deid_dataset}" --sandbox_dataset_id "${sandbox_dataset}" --mapping_dataset_id "${mapping_dataset}" --mapping_table_id "${mapping_table}" -s 2>&1 | tee -a "${LOGS_DIR}"/fitbit_log.txt
+python "${CLEAN_DEID_DIR}/fitbit_dateshift.py" --project_id "${APP_ID}" --dataset_id "${fitbit_deid_dataset}" --sandbox_dataset_id "${sandbox_dataset}" --mapping_dataset_id "${mapping_dataset}" --mapping_table_id "${mapping_table}" -s 2>&1 | tee -a "${LOGS_DIR}"/fitbit_log.txt
 
```

Comment:
And I would think we would update the phase and deid labels at this point.
```suggestion

bq update --set_label "phase:clean" --set_label "de_identified:true" "${APP_ID}":"${fitbit_deid_dataset}"
```